ksql-engine/src/main/java/io/confluent/ksql/services/SandboxedTopicDescription.java

```diff
@@ -29,13 +29,13 @@
  * <p>It adds the {@code authorizedOperations} variable. The {@code TopicDescription} does not
  * have a public constructor to add this information.
  */
-final class SandboxedTopicDescription extends TopicDescription {
+public final class SandboxedTopicDescription extends TopicDescription {
```

Comment:
Hummm.... this class is nasty.  It's extending something, but then overriding every method. Yuk!

Think I'll raise a PR to get rid of this. 

No issue with your change though.